java/flight/pom.xml

```diff
@@ -52,4 +52,17 @@
             </plugins>
         </pluginManagement>
     </build>
+
+    <profiles>
```

Comment:
does this change mean we can potentially remove this pinned version for release verification?
https://github.com/apache/arrow/blob/main/dev/release/verify-release-candidate.sh#L569

Reply:
I think so! I'll try it out